### DISCUSSION TITLE
Mission database

### DISCUSSION THREAD

**Discussion by BaptisteRC** [nasa/nos3#408] [Q&A]
Hello,
As a new user to nos3, I find difficult to locate description of the telemetries and commands that are available as standard. Is there a sort of mission database or a standard list of telemetries and commands with their basic description?
Thank you very much for any pointer or help!

**Comment by BaptisteRC** [comment]
Note: I know about the COSMOS command sender and there are some useful description there.
But it is sometimes difficult to link up what CMD with the related TM, and just in general the sequence of command for certain actions (like take a picture, download the picture, how to properly use the file system, uplink/use an onboard schedule (Absolute Time Sequence), how to configure/use the onboard telemetry table and their broadcasting rate....

**Comment by meridian-sam** [comment]
agreed that some basic documentation about simulator & MDB scope and a couple of simple examples would be very helpful for us new users.

**Comment by jlucas9** [comment]
Thanks for your suggestions! We plan to add some of this as part of issue https://github.com/nasa/nos3/issues/330
